docs/configuration/tools/bespoke.md

```diff
@@ -2,7 +2,10 @@
 
 # Bespoke Tool
 
-Add a button to the toolbar, that shows a panel which can be used to display any content that the application needs to show.
+The Bespoke tool can be used to extend the functionality of your SMK application. Use it to tailor panel content to your needs.
+
+As a simple example, you could add a list of links in HTML. As a more complex example, you could add javascript code that calls into other javascript global objects, letting you call other SMK tools or libraries used by SMK such as [Leaflet](https://leafletjs.com). Example code is in SMK's [debug/customization/bespoke](https://github.com/bcgov/smk/tree/master/debug/customization/bespoke) directory.
```

Comment:
Consider...

As a more complex example, you could add JavaScript code that accesses the SMK global object, letting you call SMK tools or libraries used by SMK such as [Leaflet](https://leafletjs.com). Example code is in the [debug/customization/bespoke](https://github.com/bcgov/smk/tree/master/debug/customization/bespoke) directory.

Reply:
Your edit is an easier read. My only concern is that it implies that all access is via the global SMK object, but we don't always have to go through it - we can call Leaflet API directly, for example 'L.LatLng()'. I vote to keep this part as-is.

Reply:
Sure, that's fine